Content.Server/StationEvents/Components/VentCrittersRuleComponent.cs

```diff
@@ -8,4 +8,10 @@ public sealed class VentCrittersRuleComponent : Component
 {
     [DataField("entries")]
     public List<EntitySpawnEntry> Entries = new();
+
+    /// <summary>
+    /// At least one special entry is guaranteed to spawn
+    /// </summary>
+    [DataField("specialEntries")]
+    public List<EntitySpawnEntry> SpecialEntries = new();
```

Comment:
Isn't this just a separate orGroup.

Reply:
guaranteed spawn is per station, not vent. I think orGroup ALWAYS spawns at least one entry from group, i tried it and got like 200 spawns on Box.